src/main/java/solvd/ermakovich/zt/service/impl/DefaultHealthIndicatorServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package solvd.ermakovich.zt.service.impl;
+
+import java.util.Map;
+import lombok.RequiredArgsConstructor;
+import lombok.SneakyThrows;
+import org.cactoos.Scalar;
+import org.springframework.stereotype.Service;
+import solvd.ermakovich.zt.domain.health.indicators.DefaultHealthIndicators;
+import solvd.ermakovich.zt.service.DefaultHealthIndicatorService;
+
+/**
+ * @author Ermakovich Kseniya
+ */
+@Service
+@RequiredArgsConstructor
+public class DefaultHealthIndicatorServiceImpl
```

Comment:
@h1alexbel it would not be clear, because there are `HealthIndicators` and `DefaultHealthIndicators` in the domain. So if i named it DefaultHealthIndicatorsService (as Default implementaiton of HealthIndicators), it would be confused.